src/dotnet-install.sh

```diff
@@ -918,15 +918,26 @@ get_http_header_wget() {
     eval $invocation
     local remote_path="$1"
     local disable_feed_credential="$2"
+    local wget_output=''
+    local wget_result=''
 
     remote_path_with_credential="$remote_path"
     if [ "$disable_feed_credential" = false ]; then
         remote_path_with_credential+="$feed_credential"
     fi
 
     wget_options="-q -S --spider --tries 5 --waitretry 2 --connect-timeout 15 "
-    wget $wget_options "$remote_path_with_credential" 2>&1 || return 1
-    return 0
+    wget_output=$(wget $wget_options "$remote_path_with_credential" 2>&1)
+    wget_result=$?
+
+    if [[ $wget_result != 0 ]] && [[ $wget_output == *"unrecognized option:"* ]]; then
```

Comment:
Can the system be localized and configured such that `wget` prints errors in another language?

Reply:
I removed the string comparison and now we rely on the exit status code.
2 means parsing error: https://www.gnu.org/software/wget/manual/wget.html#Exit-Status